### PR TITLE
New select component demos

### PR DESCRIPTION
**Pull request by shadikhani**

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -46,6 +48,90 @@
...
+    <template>
+      <style>
+        :host([theme~="custom-center"]) [part~="items"] {
+          text-align: center;
+        }
+      </style>
+    </template>
+  </dom-module>
+
+  <dom-module id="custom-select" theme-for="vaadin-item">
```
[the remark this answers is not in the document]
Comment (tomivirkki) [reply demo/index.html]: Done.

File: package.json [hunk 2]
```diff
@@ -43,5 +43,8 @@
   },
   "devDependencies": {
     "@vaadin/vaadin-component-dev-dependencies": "^3.0.0"
+  },
```
Comment (tomivirkki) [inline package.json]: Vaadin icons dependency should not be added here.

File: bower.json [hunk 3]
```diff
@@ -37,7 +37,8 @@
     "vaadin-overlay": "vaadin/vaadin-overlay#^3.2.0",
     "vaadin-lumo-styles": "vaadin/vaadin-lumo-styles#^1.1.0",
     "vaadin-material-styles": "vaadin/vaadin-material-styles#^1.1.0",
-    "vaadin-element-mixin": "vaadin/vaadin-element-mixin#^2.0.0"
+    "vaadin-element-mixin": "vaadin/vaadin-element-mixin#^2.0.0",
+    "vaadin-icons": "vaadin/vaadin-icons#^4.3.1"
```
Comment (tomivirkki) [inline bower.json]: The component itself doesn't need "vaadin-icons" runtime. It's used in the demos only so this should be moved to "devDependencies"

File: package.json (hunk 2 above)
Comment (shadikhani) [reply package.json]: Done

File: bower.json (hunk 3 above)
Comment (shadikhani) [reply bower.json]: Done

**Pull request merged**